packages/darklang/cli/local-install/main.dark

```diff
@@ -48,15 +48,16 @@ module Darklang =
       let selfUpdateIfRelevant () : Stdlib.Result.Result<Unit, String> =
         // if not installed, ignore -- nothing to self-update
 
+        // CLEANUP: what should we do if we fail to get the OS?
```

Comment:
answer: just fail (the Builtin fn raises an RTE)

Reply:
got it, thanks!